src/fidesctl/core/export.py

```diff
@@ -163,19 +164,20 @@ def generate_system_records(
         )
     ]
 
-    for system in server_system_list:
+    for system in server_resources["system"]:
```

Comment:
If nothing exists in the database, this would still exist but be an empty list - so no error returned 👍🏽 running `fidesctl export datamap` after running `fidesclt db reset -y` will just export an empty datamap (as of today anyways)